tests/manage/pv_services/test_pvc_evict_ceph_clients.py

```diff
@@ -0,0 +1,119 @@
+import logging
+import pytest
+import random
+import json
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.framework.testlib import tier2
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.node import get_worker_nodes
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+
+logger = logging.getLogger(__name__)
+
+
+@tier2
+class TestPvcEvictCephClients:
+    """
+    Test the ceph-fs PVC (RWX) mount with the following scenarios involving evict ceph-fs clients:
+    1. When two pods tries to mount the same PVC and scheduled on same node
+    2. When two pods tries to mount the same PVC and scheduled on different node
+    """
+
+    @pytest.mark.parametrize(
+        argnames=["node"],
+        argvalues=[
+            pytest.param(
+                "same",
+                marks=[
+                    pytest.mark.bugzilla("1901499"),
+                    pytest.mark.polarion_id("OCS-3985"),
+                ],
+            ),
+            pytest.param(
+                "different",
+                marks=[pytest.mark.polarion_id("OCS-3984")],
+            ),
+        ],
+    )
+    def test_pvc_evict_ceph_clients(
+        self, node, pvc_factory, teardown_factory, pod_factory
+    ):
+        worker_nodes = get_worker_nodes()
+
+        # create a RWX PVC
+        pvc_obj = pvc_factory(
+            interface=constants.CEPHFILESYSTEM,
+            access_mode=constants.ACCESS_MODE_RWX,
+            status=constants.STATUS_BOUND,
+        )
+
+        # create a pod in particular node
+        selected_node = random.choice(worker_nodes)
+        logger.info(f"creating first pod on node {selected_node}")
+        first_pod_obj = pod_factory(
+            interface=constants.CEPHFILESYSTEM,
+            pvc=pvc_obj,
+            node_name=selected_node,
+            pod_dict_path=constants.NGINX_POD_YAML,
+        )
+        helpers.wait_for_resource_state(
```

Comment:
You don't need that function. Just add to pod_factory parameters `status=constants.STATUS_RUNNING,`